org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -273,6 +273,7 @@ SsoDeveloper:
     managedPolicies:
       - 'arn:aws:iam::aws:policy/PowerUserAccess'
       - 'arn:aws:iam::aws:policy/AdministratorAccess-AWSElasticBeanstalk'
+      - !Sub "arn:aws:iam:${MasterAccount}:policy/CostExplorerAccess"
```

Comment:
Probably better to just use a [CopyValue](https://github.com/org-formation/org-formation-cli/blob/master/docs/task-files.md#copyvalue) and reference the [managed-policy template output](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/IAM/managed-policy.yaml#L30)